src/test/java/seedu/duke/utils/StateTest.java

```diff
@@ -16,22 +16,8 @@ public class StateTest {
     public void testGetSelectedModulesList_correctSelectedModuleList() {
         // validate correct module details
         Module module1 = Module.get("CS1010S");
-        assertNotNull(module1);
-        assertEquals("CS1010S", module1.moduleCode);
```

Comment:
Honestly this test class shouldn't be testing for these

Reply:
oops, missed that out thanks